src/Game.js

```diff
@@ -40,6 +42,50 @@ class Game extends Component {
     return spritesPos;
   }
 
+  /* 
+    Takes username input to save moves
+     ** Contributed by Soham Mondal <contact@sohammondal.com> **
+  */
+  userNameInputForm() {
```

Comment:
Rename this function to `renderUsernameForm`

Reply:
Done